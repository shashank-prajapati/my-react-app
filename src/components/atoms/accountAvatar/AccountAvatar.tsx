import React from 'react';
import Avatar from '@mui/material/Avatar';

type Props = {};

function AccountAvatar({}: Props) {
  return <div>
      <Avatar sx={{bgcolor:'#69A6E3'}}>A</Avatar>
  </div>;
}

export default AccountAvatar;
