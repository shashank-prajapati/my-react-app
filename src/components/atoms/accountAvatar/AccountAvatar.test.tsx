import * as React from "react";
import { render, fireEvent } from "@testing-library/react";
import AccountAvatar from "./AccountAvatar";

describe("Testing Account Avatar", () => {
  it("should match the default snapshot", () => {
    const { container } = render(<AccountAvatar initials="A" />);
    expect(container).toMatchSnapshot();
  });
});
