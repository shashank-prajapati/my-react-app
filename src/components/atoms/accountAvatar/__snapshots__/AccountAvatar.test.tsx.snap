// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Account Avatar should match the default snapshot 1`] = `
<div>
  <div>
    <div
      class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-30mkw0-MuiAvatar-root"
    >
      A
    </div>
  </div>
</div>
`;
